docker/.env.test

```diff
@@ -5,7 +5,7 @@ DB_PASSWORD=postgres
 DB_DATABASE_NAME=e2e_test
 
 # Redis
-REDIS_HOSTNAME=immich_redis_test
+REDIS_HOSTNAME=immich-redis-test
```

Comment:
Are you sure this was intended change?

Reply:
That's the hostname in the container for the test env and the e2e tests was running for 5 hours because it was wrong 😁

Reply:
Ah, okay. :D